python_scripts/mode_input_cli.py

```diff
@@ -22,17 +22,30 @@
 import lcm
 from lcm_types.ModeInput import ModeInput
 import warnings
+import argparse
 
 # Ignore deprecation warnings from ``lcm`` dependencies
 warnings.filterwarnings("ignore", category=DeprecationWarning)
 
 # Constants
 OFF = 0  # OFF behavior integer
 KILL = -1  # KILL behavior integer
-LCM_CHANNEL = "mode_input"  # must match channel name in control loop options
+DEFAULT_LCM_CHANNEL = "mode_input"
 
 
-def publish_mode(mode, print_msg="", lcm_ch=LCM_CHANNEL):
+def parse_channel():
+    """
+    """
```

Comment:
This would need a comment

Reply:
Added in 90886ddd9ff97e4bd0db4156b6bb8767f09651c6